### PR TITLE
add query load timeout config in QueryConfig

### LINKED ISSUE #2606: Load timeout for RPC from config.
I think `5000` should be a constant or feed it in the config file.

_Originally posted by @ariesdevil in https://github.com/datafuselabs/databend/pull/2600#discussion_r740842891_

### PR DESCRIPTION
**Pull request by Veeupup**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Summary about this PR
add load timeout as config in QueryConfig

## Changelog
- Improvement

## Related Issues

Fixes #2606 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
Re-run the failed job

**Comment by Veeupup** [comment]
> Re-run the failed job

It seems some tests not pass, I will check it locally

File: scripts/deploy/config/databend-query-node-1.toml [hunk 1]
```diff
@@ -29,7 +30,7 @@ http_handler_port = 8001
 cluster_id = "test_cluster"
 
 [log]
-log_level = "ERROR"
+log_level = "INFO"
```
Comment (BohuTANG) [inline scripts/deploy/config/databend-query-node-1.toml:33]: `INFO`  will prints many info to the CI log

Comment (Veeupup) [reply scripts/deploy/config/databend-query-node-1.toml]: forget to discard change..now resolved.

**Comment by drmingdrmer** [comment]
Thanks and it's a nice shot!

**Pull request merged**